### ISSUE TITLE
Problems when using percolate:synced-cron package and multi-core flag.

### ISSUE THREAD

**Issue by jamesgibson14** [meteorhacks/cluster#70]
when I add cluster to my app and add the multi-core flag, my syced-cron jobs are running once per core instead of just once like they are supposed to.

**Comment by dropfen** [comment]
you are using cluster, to run multiple instances of your application [to balance the load]

why do you think it should run once?

**Comment by jamesgibson14** [comment]
Yes, I understand that this is not a "problem" with cluster. I guess I was hoping for some feedback on how sync jobs across cores as well as machines. Is there any way to look up the id of a worker/instance so I can mark the job as "being run by worker 1" and the then stop the other from running it

**Comment by dropfen** [comment]
If you have a cron job which is running in a interval you could create an app which is doing only that thing. This would be the easiest solution if you don't need an ui for this process. However you can connect this (sub)-App to your DB and if you write something to the DB or if you change some values, the subscriptions will work as expected.

**Comment by rahlfinger** [comment]
I know this is over a year later but you can wrap your cron code in a if( !process.env.CLUSTER_WORKER_ID) check. Each cluster node has an unique ID with the master node having an ID of null.
